Webion.Firestore/Extensions/TypedDocumentSnapshot.cs

```diff
@@ -25,6 +25,18 @@ public async Task<bool> DeleteAsync()
         return result is not null;
     }
 
+    public async Task<bool> UpdateAsync<TProperty>(Expression<Func<TDoc, TProperty>> field, TProperty value)
+    {
+        var name = GetPropertyName<TDoc, TProperty>(field);
+        
+        var updateDict = new Dictionary<string, object?>();
+        updateDict.Add(name, value);
+
+        var result = await _doc.Reference.UpdateAsync(updateDict);
+        
+        return result is not null;
+    }
+
```

Comment:
Una cosa del genere non funziona?

```suggestion
    public async Task<bool> UpdateAsync<TProperty>(params (Expression<Func<TDoc, TProperty>> Field, TProperty Value) expressions)
    {
        var updateDict = new Dictionary<string, object?>();
        foreach (var e in expressions)
        {
           var name = GetPropertyName<TDoc, TProperty>(e.Field);
           updateDict.Add(name, e.Value);
        }

        var result = await _doc.Reference.UpdateAsync(updateDict);
        
        return result is not null;
    }

```

Usato come
```cs
await users.UpdateAsync(
    (u => u.Name, "Pippo"),
    (u => u.Surname, "Baudo")
);
```